l10n_it_fatturapa/models/partner.py

```diff
@@ -125,8 +128,8 @@ def _check_ftpa_partner_data(self):
                         ' generation.'
                     ) % partner.name)
 
-    @api.onchange('country_id')
-    def onchange_country_id_e_inv(self):
+    @api.onchange('country_id', 'electronic_invoice_subjected')
+    def onchange_partner_codice_destinatario(self):
```

Comment:
Come mai questa modifica? `electronic_invoice_subjected` non viene usato nel metodo

Reply:
È passato un po' di tempo, sai che sinceramente non ricordo?  
In effetti non dovrebbe esserci. :thinking: